client/src/main/proto/app/cash/backfila/client_service.proto

```diff
@@ -30,16 +30,16 @@ message PrepareBackfillRequest {
   optional bool dry_run = 5;
 }
 message PrepareBackfillResponse {
-  message Instance {
-    optional string instance_name = 1;
+  message Partition {
+    optional string partition_name = 1;
```

Comment:
it's all proto encoded so this should be safe

Reply:
Yeah I believe so.

Reply:
I think the only potential minor problem is that any of the old javascript pages might have trouble post a deploy. A simple refresh will fix it.